app/src/main/java/org/fossasia/openevent/app/qrscan/ScanQRActivity.java

```diff
@@ -48,7 +47,7 @@
 
 import static org.fossasia.openevent.app.utils.ViewUtils.showView;
 
-public class ScanQRActivity extends AppCompatActivity implements IScanQRView {
+public class ScanQRActivity extends AppCompatActivity implements IScanQRView, AttendeeCheckInFragment.OnDismissListener {
```

Comment:
Please remove this implements, 

Ummm. Please fix the changes. There was some problem in the merging I guess